ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -645,6 +645,206 @@ public final class AliasTokens {
         }
     }
 
+    // MARK: Shared Colors
+
+    public enum SharedColorsTokens: CaseIterable {
+        // Red
+        case redBackground1
```

Comment:
Maybe a question for design: why are these called `redBackground` instead of `dangerBackground`? That seems to be the proper semantic intent of these values, and alias tokens are meant to be semantic, right?

Reply:
I'll bring this up to Ian